packages/blockchain-wallet-v4-frontend/src/modals/Interest/AccountSummary/index.tsx

```diff
@@ -17,13 +17,27 @@ import { getData } from './selectors'
 import AccountSummary from './template.success'
 import Loading from './template.loading'
 
-class AccountSummaryContainer extends PureComponent<Props> {
-  state = {}
+class AccountSummaryContainer extends PureComponent<Props, State> {
+  state: State = {
+    showMoreDetails: false
+  }
 
   handleDepositClick = () => {
     this.props.interestActions.showInterestModal('DEPOSIT')
   }
 
+  handleMoreDetailsClick = () => {
+    this.setState({
+      showMoreDetails: true
+    })
+  }
+
+  handleCloseDetailsClick = () => {
+    this.setState({
+      showMoreDetails: false
+    })
+  }
+
```

Comment:
I suggest you change this to one function called `toggleMoreDetails`

```
toggleMoreDetails = () => {
  this.setState({ showMoreDetails: !this.state.showMoreDetails })
}
```

Reply:
good call out, thanks.